vector/src/main/java/im/vector/app/features/settings/devices/v2/othersessions/OtherSessionsFragment.kt

```diff
@@ -57,6 +59,7 @@ class OtherSessionsFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        views.otherSessionsToolbar.title = getString(args.titleResourceId)
         setupToolbar(views.otherSessionsToolbar).allowBack()
```

Comment:
I think you can call `setTitle(args.titleResourceId)` instead just right after `setupToolbar()` to configure the toolbar.

Reply:
Cool, done.